docker/transformers-all-latest-gpu/Dockerfile

```diff
@@ -68,6 +68,9 @@ RUN python3 -m pip install --no-cache-dir g2p-en
 # For Some bitsandbytes tests
 RUN python3 -m pip install --no-cache-dir einops
 
+# For SmolVLM tests
+RUN python3 -m pip install --no-cache-dir num2words
```

Comment:
we don't need this as we install `-e ./transformers[dev,onnxruntime]` (in an earlier line) and `dev` includes `all`

Reply:
good point, removing it

Do you want me to repeat the image building process, or is it okay to merge if CI gets green?